README.md

```diff
@@ -778,6 +786,36 @@ Show the detail statistic of specific entity type. You can specify which part of
 
 For example, `/lifetime creeper` shows all statistic of creeper in detail, and `/lifetime creeper removal` only shows removal statistic of creeper in detail 
 
+# Scarpet
+
+## Functions
+
+### `register_block(pos)`
+
+Registers a block position to be tracked with events by scarpet. It is added to a global tracker (which you can view with
+`registered()`). Blocks in this list will trigger events in `__on_block_event` if they occur, independently of the loggers.
+Returns `true` if block was not previously in list, and `false` if it was.
+
+### `deregister_block(pos)`
```

Comment:
does `unregister` sound better than `deregister`? never heard of `deregister` before

Reply:
k